static/css/form.css

```diff
@@ -210,6 +210,10 @@ along with BeansBooks; if not, email info@beansbooks.com.
 	width: 312px;
 }
 
+.form .row.column-10 > span.subway-dance {
+	width: 400px;	
```

Comment:
Can we get this changed to 404 px?  It needs to be a bit more than 400 for the other rows to line up correctly.